frontend/src/main/scala/bloop/Bloop.scala

```diff
@@ -1,25 +1,29 @@
 package bloop
 
-import bloop.cli.{Commands, ExitStatus}
+import bloop.cli.{CliOptions, Commands, ExitStatus}
+import bloop.cli.CliParsers.{inputStreamRead, printStreamRead, OptionsParser, pathParser}
 import bloop.engine.{Build, Exit, Interpreter, Run, State}
 import bloop.engine.tasks.CompileTasks
 import bloop.io.AbsolutePath
 import bloop.io.Timer.timed
 import bloop.logging.BloopLogger
 import jline.console.ConsoleReader
+import caseapp.{CaseApp, RemainingArgs}
 
 import scala.annotation.tailrec
 
-object Bloop {
+object Bloop extends CaseApp[CliOptions] {
```

Comment:
Bloop is supposed to emulate `CaseApp`'s logic without extending it, so this was on purpose. Why the change?